app/uk/gov/hmrc/agentsubscription/model/AgencyDetails.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.agentsubscription.model
 import play.api.libs.functional.syntax._
 import play.api.libs.json.{ Json, Reads, _ }
 
-case class BusinessDetails(tradingName: String, businessAddress: OverseasBusinessAddress)
+case class TradingDetails(tradingName: String, tradingAddress: OverseasBusinessAddress)
 case class BusinessContactDetails(businessTelephone: String, businessEmail: String)
 
 case class AgencyDetails(
```

Comment:
I wonder if all the other modle classes other than `AgencyDetails` should be in a separate file as `AgencyDetails` is shared between UK and Overseas whilst all the other model classes are all overseas specific.

Reply:
Actually, ignore that. It's used only for overseas agents